pdns/ws-api.cc

```diff
@@ -159,15 +159,29 @@ void apiServerStatistics(HttpRequest* req, HttpResponse* resp) {
   if(req->method != "GET")
     throw HttpMethodNotAllowedException();
 
+  Json::array doc;
+  string name = req->getvars["statistic"];
+  if (!name.empty()) {
+    auto stat = productServerStatisticsFetch(name);
```

Comment:
would a `serverStatisticsFetch` function that uses `#ifdef RECURSOR` with an `#else` be more readable than this indirection?

Reply:
I actually find the current way cleaner than having conditional includes and code blocks.